### ISSUE TITLE
Clean install errors

### ISSUE THREAD

**Issue by Dutchboy** [TeamMaestro/angular-native-seed#13]
Hi!!
I just did a
```
git clone
cd angular-native-seed
npm i
cd nativescript
npm i
```
And on the last `npm i` it errored at `nativescript-dev-webpack` `post install`. I'm seeing that the whole subdir under` npm_modules` doesn't exist. I also see not-so-up-to-date versions in the package.json, maybe thats the cause?

I did it on both a Windows10 and a macOS (Sierra)  machine, same results.

**Comment by db3dev** [comment]
Thanks, I found the issue was that the script is trying to create a file in the nonexistent app directory during install. If you create an empty app folder in your nativescript folder your install will be successful. I'll update master to include this folder.

**Comment by sean-perkins** [comment]
@db3dev Also note the `symlink` script has a flaw on Windows that's addressed in #12. The script references an invalid file name when it was copied from Nathan's seed.

**Comment by db3dev** [comment]
#15 solves the clean install issue where nativescript-dev-webpack was not able to successfully install.
